spring-batch-core/src/main/java/org/springframework/batch/core/partition/support/Partitioner.java

```diff
@@ -30,7 +31,7 @@
  * @author Taeik Lim
  * @since 2.0
  */
-@FunctionalInterface
+// @FunctionalInterface
```

Comment:
can you remove `//`? (maybe we should remove `@FunctionalInterface` cause we have 2 method

Reply:
I modified it as you reviewed. Thank you!